src/client/Microsoft.Identity.Client/WsTrust/WsTrustWebRequestManager.cs

```diff
@@ -26,8 +27,29 @@ public WsTrustWebRequestManager(IHttpManager httpManager)
         }
 
         /// <inheritdoc/>
-        public async Task<MexDocument> GetMexDocumentAsync(string federationMetadataUrl, RequestContext requestContext)
+        public async Task<MexDocument> GetMexDocumentAsync(string federationMetadataUrl, RequestContext requestContext, string federationMetadataFilename)
```

Comment:
Is `federationMetadataFilename` a required field or can it be optional too?

Reply:
probably should be optional